R/convertMT.R

```diff
@@ -0,0 +1,103 @@
+#' @title Convert migration times to effective mobility
+#'
+#' @description
+#'
+#' `convertMtime` performs effective mobility scale transformation of CE(-MS) 
+#' data, which is used to overcome variations of the migration times, caused by 
+#' differences in the Electroosmotic Flow (EOF) between different runs. 
+#' In order to monitor the EOF and perform the transformation, neutral or 
+#' charged EOF markers are spiked into the sample before analysis. The 
+#' information of the EOF markers (migration time and effective mobility) will 
+#' be then used to perform the  effective mobility transformation of the 
+#' migration time scale. 
+#'
+#' @param x `numeric` vector with migration times in minutes.
+#'
+#' @param rtime `numeric` vector that holds the migration times (in minutes) of 
+#' either one or two EOF markers in the same run of which the migration time is 
+#' going to be transformed.
+#' 
+#' @param mobility `numeric` vector containing the respective effective mobility
+#' (in in mm^2 / (kV * min)) of the EOF markers. If two markers are used, one is
+#' expected to be the neutral marker, i.e. having a mobility of 0. 
+#' 
+#' @param tR `numeric` a single value that defines the time (in minutes) of the 
+#' electrical field ramp. The default is 0. 
+#' 
+#' @param U `numeric` a single value that defines the voltage (in kV) applied. 
```

Comment:
I would maybe mention that this parameter is only used if convertion is done on a single marker (i.e. length of `rtime` and `mobility` is one).